src/kudu/tserver/consensus_service.cc

```diff
@@ -430,6 +491,11 @@ void ConsensusServiceImpl::RunLeaderElection(const RunLeaderElectionRequestPB* r
   shared_ptr<RaftConsensus> consensus;
   if (!GetConsensusOrRespond(tablet_manager_, resp, context, &consensus)) return;
 
+  if (!CheckRaftRpcTokenOrRespond("RunLeaderElection", req, resp, context,
```

Comment:
Because we do nothing on the response. We can technically add it for completeness, but today if you look at the call in consensus_peers, you'll see that we don't even have a callback to handle the response we receive, so even if we set it, no one will read it.